CalPatRec/inc/ChannelID.hh

```diff
@@ -0,0 +1,19 @@
+#ifndef __CalPatRec_inc_ChannelID_hh__
+#define __CalPatRec_inc_ChannelID_hh__
+
+namespace CalPatRec {
```

Comment:
this is an internal for the algorithm helper class which returns hit indices ordered in Z.  This functionality is not provided by the StrawId class.

Reply:
at the time when the code has been developed, the StrawId interface was not providing an ordering of the faces in assending global Z. As you already mentioned, it might be a good idea to integrate this functionality into the StrawId class

Reply:
I will start an issue and assign it for resolution after this PR